lib/openhab/dsl/timers/manager.rb

```diff
@@ -72,7 +72,7 @@ def delete(timer)
           @timers.delete(timer)
           if timer.respond_to? :id
             @timer_ids[timer.id]&.delete(timer)
-            @timer_ids.delete(timer.id) unless @timer_ids[timer.id].any?
+            @timer_ids.delete(timer.id) if @timer_ids[timer.id] && @timer_ids[timer.id].empty?
```

Comment:
Thanks for finding this! This was my bad from a previous PR. We could just check for `@timer_ids[timer.id]` not being nil once, i.e.

```ruby
          if timer.respond_to?(:id) && @timer_ids[timer.id]
            @timer_ids[timer.id].delete(timer)
            @timer_ids.delete(timer.id) if @timer_ids[timer.id].empty?
```